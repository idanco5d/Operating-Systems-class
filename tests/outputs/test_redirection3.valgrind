==9447== Memcheck, a memory error detector
==9447== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==9447== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==9447== Command: /mnt/c/Users/illus/os/hw/1/smash
==9447== Parent PID: 9442
==9447== 
==9447== 
==9447== FILE DESCRIPTORS: 4 open (3 std) at exit.
==9447== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_redirection3.valgrind
==9447==    <inherited from parent>
==9447== 
==9447== 
==9447== HEAP SUMMARY:
==9447==     in use at exit: 0 bytes in 0 blocks
==9447==   total heap usage: 158 allocs, 158 frees, 83,905 bytes allocated
==9447== 
==9447== All heap blocks were freed -- no leaks are possible
==9447== 
==9447== For lists of detected and suppressed errors, rerun with: -s
==9447== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
