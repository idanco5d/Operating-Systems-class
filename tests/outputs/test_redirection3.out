smash> smash> smash> smash> smash> smash pid is 9663
smash pid is 9663
smash pid is 9663
smash pid is 9663
smash> smash> smash pid is 9663
smash> smash: sending SIGKILL signal to 0 jobs:
